src/templates/default_procedure/app.py

```diff
@@ -5,7 +5,7 @@
 from snowflake.snowpark import Session
 
 
-def hello() -> str:
+def hello(session: Session) -> str:
```

Comment:
It probably won't work because you have changed `requirements.txt` for function, not for procedure.